internal/evaluator/conftest_evaluator.go

```diff
@@ -161,13 +161,33 @@ func (c *conftestEvaluator) addDataPath(spec *ecc.EnterpriseContractPolicySpec)
 	}
 
 	if spec != nil {
+		type policyConfig struct {
```

Comment:
I think this is caused by a slight disconnect between what is in CR vs what the ec-policies expect. In CR we have:

```yaml
spec:
  exceptions:
    nonBlocking:
  config:
    excludeRules:
    includeRules:
    collections:
```

And ec-policy needs:

```yaml
config:
  policy:
    non_blocking_checks:
    excludeRules:
    includeRules:
    collections:
```

Anyhow, this goes away and the `spec.configuration` from ECP CR gets used for `config` for ec-policies once we get rid of "non blocking" support. Am I correct @robnester-rh? Perhaps we could create a `TODO` comment here just in case.

Reply:
Yes. Once we have everything synced, we can update this further.

Reply:
OK, thanks for the explanation!